packages/app-accounts/src/md/basic.md

```diff
@@ -0,0 +1,40 @@
+# Account
+
+An account is identified by its public address on the network. The network will **not** know about the name you give to this account in this app. It is totally fine to give away this address, this is also the only information needed to receive funds.
+
+# Security
+
+The public address is generated from a private key, itself generated from a seed or a mnemonic phrase. The seed or the mnemonic phrase should **never be shared with anybody** as they give access to your funds. It must be stored securely.  
+The password needed to create an account is used to encrypt your private key. You must choose a strong and unique password.  
+This password is also used to encrypt the private key in the backup file downloaded upon account creation. Thanks to this file together with your account password, you can recover your account.
+
+# Account recovery
+
+You can recover an account from its:
+- seed or mnemonic:  
+  From the "Create account" tab, type your seed or mnemonic in the associated field.
+
+- backup file (also called JSON keystore file) and the account's password:
+  From the "Restore account" tab. Upload your backup file and type in the password associated.
+
+# Minimum allowed balance
+
+Accounts with a balance lower than the minimal amount, 100 mili Unit as of writing are considered as inexistent for the network. If an account's balance ever drops below this amount, it is removed from the network. In this application, it will still be visible, but with a balance of 0.
```

Comment:
```suggestion
Accounts with a balance lower than the minimal amount, 100 milliDOTs as of writing are considered as nonexistent for the network. If an account's balance ever drops below this amount, it is removed from the network. In this application, it will still be visible, but with a balance of 0.
```

Units would be better for Substrate chains but maybe confusing for people needing help with Polkadot?

Reply:
Hmm, but the help, like the whole interface,  is not only for Polkadot, but also for Substrate chains. So keeping it more generic with 'Unit' makes more sense. Having the help mentioning DOTs on Embric Elm and other Substrate chains is equally confusing (and wrong).

Reply:
Right, since Polkadot users might be the ones to be the most confused (Substrate users know about Polkadot, but Polkadot users might not know about Substrate), I've catered for both using " miliUnits (miliDOTs for Polkadot)"